comic_identity_engine/jobs/tasks.py

```
@@ -0,0 +1,857 @@
+
+                    if platform in existing_by_source:
+                        mappings_updated += 1
+                    else:
+                        logger.info(
+                            "No existing mapping found for platform",
+                            operation_id=operation_id,
+                            issue_id=issue_id,
+                            platform=platform,
+                        )
+
+                except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)